docs/wiki/feature/atragmx.md

```diff
@@ -12,21 +12,68 @@ version:
-Short description of sub-feature 2.
+Horus ATragMX software considers atmospheric conditions, gun data, ammunition, range, speed and muzzle velocity to calculate precise aiming solutions with "come-up" results – and even accounts for Coriolis and spin drift effects. ATragMX, loaded on a handheld computer made by TDS Recon, is easy to use and lightning-fast. The Recon meets the rigorous MIL-STD-810F military standard for drops, vibration, humidity, altitude and extreme temperatures.
 
+## 2. Usage
 
-## Usage
+### 2.1 Example with M14 and default 7.62mm 20Rnd Mag
 
-Short overview of how to use the feature, e.g. menu options, key bindings,
-instructions. May not apply to all modules.
+**Start of the mission:**
+ - Open the Range Card and check the cartridge, the zeroed distance, the rifle twist, 
+the Muzzle Velocity at 15°C and the Bore Height at the end.
 
+ img src="{{ site.baseurl }}/img/wiki/feature/atragmx1.jpg" width="431" height="600" alt="RangeCard" /
 
-## Dependencies
+ - Open the AtragMx and the `Atmsphr` column, select `Default` and `Done`.
+ - `Open Gun` the 7.62x51mm M80 in the `GunList`.
+ - Select `E` (English unit) at the top right.
+ - Open the `Gun` column, check and update the `Bore`, the `Rifle Twist` and `Done`.
+ - Select `M` (Metric unit) at the top right.
+ - Open the `Gun` column, check and update the `Muzzle Velocity`, the `Zero Range` and `Done`. 
+ - *The Muzzle Velocity Table will be automatically updated.*
+ - Optionally, `Save Gun` and `Done` in the `GunList`.
+ 
+**In position:**
+ - Update the `Atmsphr` column with the Kestrel and `Done` (Check the new Muzzle Velocity in the `Gun` column).
+ - Update the `Target` column.
+ - Apply the vertical and horizontal elevations on the scope.
+ - Control the breath and press.
+ 
+### 2.2 Example with Truing tool
+ 
+**Start of the mission:**
+ - Select `Drag Coef Table` in the `Options` menu.
+ - Add the `ZR` and the `C1` (`Gun` column) in the table and `Done`.
+  
+**In position:**
+ - Open the `Truing Drop` in the `Options` menu.
+ - Add the actual `Target Range` in the `SUPER` column and `Calc`.
+ - Add the same `Target Range` in the `SUB` column  and `Calc`.
+ - Apply the actual scope elevation in the `Drop` field and `Calc`.
+ - `Accept` the new `C1`, `Gun` column and `Elev` are updated.
+ - *The Drag Coefficient Table will be automatically updated.*
+ - Optionally, `Save Gun` and `Done` in the `GunList`.
+ 
+ img src="{{ site.baseurl }}/img/wiki/feature/atragmx2.png" width="1127" height="600" alt="Calculation" /
+ 
+ - If a new `Target Range` is applied in the `Target` column, select `Drag Coef Table` in the `Options` menu and `Done`.
+ - The ballistic coefficient `C1` and the elevation `Elev` will be recalculated.
+ 
+img src="{{ site.baseurl }}/img/wiki/feature/atragmx3.png" width="1123" height="600" alt="Interpolation" /
+ 
+ ### 3. Official Manual and Horus Videos
+
+[Official Manual](https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf)
```

Comment:
Sorry jonpas but i don't understand exactly what you want. 
Dillinger and a french tuto give me this way for a link.
You prefer something like that ? Without `[Official Manual]`?
`- <https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf>`

Or something like that maybe ?
`- [Official Manual]: <https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf>`

(My apologies for my mistakes)